CHANGELOG.adoc

```diff
@@ -25,6 +25,9 @@ endif::[]
 * improvement: add multiple caches for accelerating available container count calculation （#667）
 * improvement: RecordContext now exposes lastFailureReason (#725)
 
+=== Fixes
+* fix: remove lingeringOnCommitWouldBeBeneficial and unused imports
+
```

Comment:
Could you please move this higher up - merge into the Fixes subsection above? And regenerate Readme again once its done.

Reply:
@rkolesnev thanks for pointing out, changed accordingly.